indico/modules/events/registration/templates/management/_reglist.html

```diff
@@ -24,7 +24,13 @@
                             <th class="i-table">{% trans %}Full name{% endtrans %}</th>
 
                             {% for item in static_columns %}
-                                <th class="i-table" data-sorter="text">{{ item.caption }}</th>
+                                {% if not item.get('filter_only') %}
+                                    {% if item.id == 'tags_present' %}
+                                        <th class="i-table" data-sorter="text">{% trans %}Tags{% endtrans %}</th>
+                                    {% else %}
```

Comment:
Why is this needed btw? The `.title` should be correct

Reply:
Thanks for spotting that, it's not needed anymore. Before, the custom filter showed `Registration tags` and the table was just `Tags`. Now they're the same so it's redundant.